Sources/Testing/CMakeLists.txt

```diff
@@ -31,10 +31,11 @@ add_library(Testing
   Events/Recorder/Event.JUnitXMLRecorder.swift
   Events/Recorder/Event.Symbol.swift
   Events/TimeValue.swift
-  ExitTests/ExitCondition.swift
   ExitTests/ExitTest.swift
-  ExitTests/ExitTestArtifacts.swift
+  ExitTests/ExitTest.Condition.swift
+  ExitTests/ExitTest.Result.swift
   ExitTests/SpawnProcess.swift
+  ExitTests/StatusAtExit.swift
```

Comment:
You mentioned in the PR description this type may be used independently of exit tests, so I'm wondering if the file ought to be placed in a different directory to emphasize that.